src/styles/blocks/main/product-card/card.scss

```diff
@@ -0,0 +1,64 @@
+.card {
+  width: $card-width;
+  padding: 32px $spacing $spacing;
```

Comment:
Also in this file generally try to set values divisible by 5. It's easier for the sake of figuring out and adjusting the positions (except borders ofc)

Reply:
Should i forget about passing the tests?

Reply:
Generally try to please the tests, if you are significantly breaking them (your work differs from the designs a lot) then conform to the tests requirements. Your current implementation meets the task requirements and looks very good so I wouldn't bother that much with the tests here, in real life they always can be adjusted to represent actual state of things (not cheated ofc)